modules/user-guide/pages/installing-redhat-style-for-vale.adoc

```diff
@@ -0,0 +1,72 @@
+// Metadata for Antora
+:navtitle: Installing the RedHat style
+:keywords: installing-redhat-style-for-vale, antora
+:description: Installing the Red Hat style for Vale
+// End of metadata for Antora
+:_module-type: CONCEPT
+[id="installing-redhat-style-for-vale"]
+= Installing the Red Hat style for Vale globally
+
+You can install the `RedHat` style for Vale globally.
+
+[IMPORTANT]
+====
+Some product documentation repositories, such as `openshift-docs`, have an existing `.vale.ini` file.
+When you run `vale` on content files in the repository directory, the `<repository>/.vale.ini` file overrides the `$HOME/.vale.ini` file. For more information, see link:https://vale.sh/docs/topics/config[Vale configuration reference].
```

Comment:
Is it worthwhile to say something like, run `vale sync` in your repo directory to download the Vale configuration relative to that repo. 
This is another issue I think people are falling over